package.json

```diff
@@ -84,12 +84,12 @@
     "vite-tsconfig-paths": "^4.3.2"
   },
   "dependencies": {
-    "classnames": "^2.2.6"
+    "@testing-library/dom": "^10.4.1",
+    "classnames": "^2.2.6",
+    "lodash": "^4.17.21"
```

Comment:
The two new packages should have been saved as `devDependencies`?

Thanks for spotting `lodash` was missing from **.storybook/manager.ts**